cobra/io/sbml.py

```diff
@@ -58,8 +59,9 @@ class CobraSBMLError(Exception):
 # ----------------------------------------------------------
 # Defaults for writing SBML
 # ----------------------------------------------------------
-LOWER_BOUND = -1000
-UPPER_BOUND = 1000
+config = cobra.Configuration()
+LOWER_BOUND = config.lower_bound
```

Comment:
My bad, actually `config.lower_bound` should be used instead of `LOWER_BOUND` everywhere. Otherwise this will be set only at import time and later changes to the configuration values will have no effect.

Reply:
done